packages/ns-api/files/ns.reverseproxy

```diff
@@ -15,38 +15,21 @@ from nethsec import utils
 from nethsec.utils import ValidationError
 
 
-def __write_changes(uci):
-    # first commit allowing nginx-proxy to generate location files
-    e_uci.commit('nginx')
-    # generate location files
-    try:
-        subprocess.run(['nginx-proxy'], check=True)
-    except subprocess.CalledProcessError:
-        raise RuntimeError('Failed to generate nginx configuration')
-    # append _lan proxy to nginx if exists and not already included otherwise remove it
-    includes = list(e_uci.get('nginx', '_lan', 'include', list=True))
-    if os.path.isfile('/etc/nginx/conf.d/_lan.proxy'):
-        if 'conf.d/_lan.proxy' not in includes:
-            includes.append('conf.d/_lan.proxy')
-    else:
-        if 'conf.d/_lan.proxy' in includes:
-            includes.remove('conf.d/_lan.proxy')
-    e_uci.set('nginx', '_lan', 'include', includes)
-    # commit changes to include _lan proxy
-    uci.commit('nginx')
-    # reload nginx
-    try:
-        subprocess.run(['/etc/init.d/nginx', 'reload'], check=True)
-    except subprocess.CalledProcessError:
-        raise RuntimeError('Failed to reload nginx')
-
-
 def __create_location(uci, uci_server, location, proxy_pass):
     location_id = utils.get_random_id()
     uci.set('nginx', location_id, 'location')
     uci.set('nginx', location_id, 'uci_server', uci_server)
     uci.set('nginx', location_id, 'location', location)
-    uci.set('nginx', location_id, 'proxy_pass', proxy_pass)
+    prefix = proxy_pass.split('://')[0]
+    address = proxy_pass.split('://')[1]
+    # checking if address is a domain or an ip4/6 address
+    if address.count('.') == 3 or address.count(':') == 7:
```

Comment:
This could match also a host named "my.host.org".
I'd use something like:
```python
try:
    ipaddress.ip_address(address)
   e_uci.set('nginx', location_id, 'proxy_pass', proxy_pass)
except:
   e_uci.set('nginx', location_id, 'resolver', '127.0.0.1')
   e_uci.set('nginx', location_id, 'set', f'$upstream {address}')
   e_uci.set('nginx', location_id, 'proxy_pass', f'{prefix}://$upstream')
```

Otherwise we could always set the resolver, probably it should work also with IP addresses.